woocommerce/woocommerce-ios
# Dark Mode: update colors for login flow

Login flow for #1638 

- [x] ⚠️Update `Podfile` after https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/173 is merged and released ⚠️ 

## Changes

- Most of the changes are in the `WordPressAuthenticator` PR: TODO after creating the PR
- Added 2 new image assets for login flow and passed these two images in `WordPressAuthenticatorDisplayImages` to initialize `WordPressAuthenticator`:
  - `loginMagicLinkImage`
  - `loginSiteAddressInfoImage`
- Updated colors for the login with site address info alert in `AppDelegate`

## Testing

- Log out of the app
- Tap "Log in with Jetpack"
- Tap "Enter the address..."
- Tap "Need help finding your site address?" --> the alert should look as expected in Light and Dark modes (note that some colors are cached when switching Light/Dark mode on the same screen during testing. Because this alert UI component is in another shared library `WordPressUI` and most of the users probably wouldn't switch Light/Dark mode on this particular screen, it might make sense to skip this edge case for now)
- Tap "OK" to dismiss the alert
- Enter a valid site address and tap "Next" --> the icon of the site address should have the expected color
- Navigate back to the home screen
- Tap "Log in with Jetpack"
- Enter a non-a8c email address and tap "Next"
- Tap "Send Link" --> the image and the "Enter your password instead." button should have the expected colors

## Example screenshots

Note for @kyleaparker: for the site address help alert Light mode, the lightest background color behind the header image I could find in our color palette is system gray6 but it still looks darker than the design. Do you know if there is a closer color?

Screen | Dark | Light
-- | -- | --
Site address help alert | ![Simulator Screen Shot - iPhone 11 - 2019-12-23 at 17 01 34](https://user-images.githubusercontent.com/1945542/71350555-78d44000-25ac-11ea-8e4f-b519f9ba19e8.png) | ![Simulator Screen Shot - iPhone 11 - 2019-12-23 at 17 01 44](https://user-images.githubusercontent.com/1945542/71350556-796cd680-25ac-11ea-8662-6ea062de3604.png)
After entering site address | ![Simulator Screen Shot - iPhone 11 - 2019-12-23 at 17 01 23](https://user-images.githubusercontent.com/1945542/71350490-678b3380-25ac-11ea-8f50-ee90a4452a75.png) | ![Simulator Screen Shot - iPhone 11 - 2019-12-23 at 17 01 14](https://user-images.githubusercontent.com/1945542/71350445-53473680-25ac-11ea-8727-fe6e6f1cbbfe.png)
Magic link | ![Simulator Screen Shot - iPhone 11 - 2019-12-23 at 17 00 10](https://user-images.githubusercontent.com/1945542/71350412-3dd20c80-25ac-11ea-8984-ffe2000b6f7f.png) | ![Simulator Screen Shot - iPhone 11 - 2019-12-23 at 17 00 33](https://user-images.githubusercontent.com/1945542/71350413-3dd20c80-25ac-11ea-823b-befb48bb33da.png)

Update release notes:
